catroid/src/androidTest/java/org/catrobat/catroid/uitest/ui/dialog/NewSpriteDialogTest.java

```diff
@@ -67,20 +63,6 @@ protected void tearDown() throws Exception {
 	}
 
 	public void testNewSpriteDialogStep1() throws Exception {
-		UiTestUtils.getIntoSpritesFromMainMenu(solo);
```

Comment:
I have removed the ui tests for now. If I've understood the priority of this PR correctly, there was no time to add new espresso tests for the functionality. @cheidenreich, is that so? or should I add before we merge it?